src/ngx_http_set_hashed_upstream.c

```diff
@@ -1,27 +1,61 @@
 #ifndef DDEBUG
 #define DDEBUG 0
 #endif
+
 #include "ddebug.h"
 
 #include "ngx_http_set_hashed_upstream.h"
 
 
 ngx_uint_t
-ngx_http_set_misc_apply_distribution(ngx_log_t *log, ngx_uint_t hash,
-    ndk_upstream_list_t *ul, ngx_http_set_misc_distribution_t type)
-{
+ngx_http_set_misc_apply_distribution(ngx_log_t *log, ngx_uint_t hash, ngx_http_set_hashed_upstream_conf *hashConf,
+                                     ngx_http_set_misc_distribution_t type) {
     switch (type) {
-    case ngx_http_set_misc_distribution_modula:
-        return (uint32_t) hash % (uint32_t) ul->nelts;
+        case ngx_http_set_misc_distribution_modula:
+            return (uint32_t) hash % (uint32_t) hashConf->ul->nelts;
+        case ngx_http_set_misc_consistent_hash:
+            return ngx_http_set_misc_bsearch(hashConf->hashNodes, hashConf->nodeLen, hash) %
```

Comment:
We usually do not use uppercase in variable name. `hashNodes` should be `hash_nodes`.